release.config.cjs

```diff
@@ -0,0 +1,83 @@
+const transform = require("./release.utils.cjs");
+module.exports = {
+    branches: [
+        "+([0-9])?(.{+([0-9]),x}).x",
+        "master",
+        "next",
+        "next-major",
+        {
+            name: "beta",
+            prerelease: true,
+        },
+        {
+            name: "alpha",
+            prerelease: true,
+        },
+        {
+            name: "development",
+            prerelease: true,
+        },
+    ],
+    repositoryUrl: "git@github.com:deriv-com/deriv-utils.git",
+    plugins: [
+        [
+            "@semantic-release/commit-analyzer",
+            {
+                releaseRules: [
+                    {
+                        type: "feat",
+                        release: "minor",
```

Comment:
These are the release rules, which dictates what version it should be bumped to. The type of releases `feat`, `build`, `chore` are all checked within your commits.

So if you commited something like `chore: updated time.ts` and merged it to `master` branch, then the version will be patched from `0.0.11` to `0.0.12`, likewise if you commited something like `feat: added language utilites` and merged it to master, the version would be bumped minor from `0.0.11` to `0.1.11`